src/main/java/com/pmarko09/medical_clinic_proxy/controller/DoctorController.java

```diff
@@ -17,8 +15,9 @@ public class DoctorController {
 
     private final DoctorService doctorService;
 
-    @GetMapping("/{email}/available-visits")
-    public List<VisitDto> getAvailableVisitsForDoctor(@PathVariable String email) {
-        return doctorService.getAvailableVisitsForDoctor(email);
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
nie musimy dawać ResponseStatus OK poniewaz to jest defaultowy status odpowiedzi HTTP

Reply:
usuniete